react/src/lib/components/SubsurfaceViewer/components/ViewFooter.tsx

```diff
@@ -1,23 +1,19 @@
 import React from "react";
-import { createStyles, makeStyles, Theme } from "@material-ui/core";
 import PropTypes from "prop-types";
+import { styled } from "@mui/system";
 
-const useStyles = makeStyles((theme: Theme) =>
-    createStyles({
-        viewFooter: {
-            position: "absolute",
-            bottom: theme.spacing(0),
-            right: theme.spacing(2),
-            zIndex: 999999,
+const StyledViewFooter = styled("div")(({ theme }) => ({
```

Comment:
Can't you simple call that ViewFooter, whitout the need of the wrapper on line 22 ?

Reply:
Not sure how to do that while still capturing the `theme`.